enhancements/upgrade-bundle.md

```diff
@@ -0,0 +1,138 @@
+---
+title: upgrade-bundle
+authors:
+  - "@rashmigottipati"
+reviewers:
+  - "@joelanford"
+  - "@estroz"
+  - "@shawn-hurley"
+  - "@jmrodri"
+approvers:
+  - TBD
+creation-date: 2020-06-11
+last-updated: 2020-06-11
+
+---
+
+# Upgrade Bundle
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+1. How do we handle multiple subscriptions? 
+2. Should we explicitly pass new version number as a flag or can we get it from bundle image?
+    * Based on discussions, can extract the version number from bundle image. 
+3. How do we verify that the upgrade is successful 
+4. Is it valid to assume that no previous version of the operator is already running? 
+
+## Summary
+
+Enhance the Operator SDK to add a new subcommand `run bundle-upgrade` that performs an operator upgrade using OLM.
+
+## Motivation
+
+Testing operator upgrades remains difficult today due to the number of steps to prepare for an upgrade and the lack of tooling that automates those steps. Having high quality Operators is crucial to the success of the Operator Framework and thereby operator developers should have a tool to use to perform an operator upgrade.
+
+### Goals
+
+Provide a command line that upgrades an operator version using OLM.
+
+### Non-Goals
+
+## Proposal
+
+The Operator SDK currently supports running an operator with OLM using the
+`run` command with the `--olm` flag. 
+This enhancement will add a new subcommand, `bundle-upgrade`,
+to the `run` command. This new command, `run bundle-upgrade`, will perform the upgrade of an operator from a previous version to a new specified version.
+
+Currently, the run command has different subcommands: 
+
+ * `run local`
+ * `run packagemanifests`
+ * `run bundle` - currently being implemented, design document is covered by [run bundle enhancement](https://github.com/operator-framework/enhancements/blob/master/enhancements/run-bundle.md)
+
+#### Command Line Interface
+The new command line will look as follows:
+
+```
+operator-sdk run bundle-upgrade (alias: bu) <bundle-image> --from-index=<> --starting-csv=<> [--channel=<>] [--operator-namespace=<>] [--install-mode=<>]
+```
+
+* Positional arguments:
+    * `bundle-image`
+* Required flags:
+    * `--index-image` 
+    * `--starting-csv`
+* Optional flags:
+    * `--channel` 
+    * `--operator-namespace`
+    * `--install-mode`
+
+* `<bundle-image>` is a positional argument that specifies the bundle image that we want to upgrade to
+* `--index-image=<>` is a required flag, which contains the operator we’re upgrading from
+* `--starting-csv=<>` is a required flag, which specifies the first version of the CSV that OLM will install as part of the subscription for a given package. 
```

Comment:
There's a lot of information packed into these flag descriptions. Can you write a paragraph or two about how you'll use these values in concert in an implementation details section?

Reply:
From my understanding of this comment, it's to handle scenarios of simple (requiring minimal flags)/complex (providing all the flags) example usages.
I just expanded on it in the `Example UX scenarios` section. Let me know if this is what you were expecting. If not, happy to discuss further.